core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/HedvigMaterial3ColorScheme.kt

```diff
@@ -115,17 +134,39 @@ val ColorScheme.onTypeElement: Color
   @Composable
   get() = LocalHedvigMaterial3ColorScheme.current.onTypeElement
 
-val ColorScheme.typeSignalContainer: Color
+val ColorScheme.typeContainer: Color
   @ReadOnlyComposable
   @Composable
   get() = LocalHedvigMaterial3ColorScheme.current.typeContainer
 
-val ColorScheme.onTypeSignalContainer: Color
+val ColorScheme.onTypeContainer: Color
   @ReadOnlyComposable
   @Composable
   get() = LocalHedvigMaterial3ColorScheme.current.onTypeContainer
 // endregion
 
+// region info colors
+val ColorScheme.infoElement: Color
```

Comment:
Nice that you can define own colors :)

Reply:
Yep, and can use them like `MaterialTheme.colorScheme.infoElement` which makes it feel like it was part of the material theme all along.